src/handbook/development/releases/process.md

```diff
@@ -3,85 +3,161 @@ navTitle: Release Process
+Follow these steps to run the scripts:
 
-### Next Steps
+1. Checkout the `flowforge/admin` repository if you do not already have it.
+   Ensure you have the latest with a `git pull`.
+2. In the *parent* directory to where you have the `admin` checked out, run the
+   following command, replace `1.x.y` with the proper release version.
+   ```bash
+   ./admin/checkout-release 1.x.y
+   ```
+   This will create a directory called `release-1.x.y` and checkout all of the releasable
+   repositories under it.
+   **NOTE:** If you do not have a global git configuration, you will need to set `user.name` and `user.email`
+   in each of those repositories.
+3. Within the release directory run:
+   ```bash
+   cd release-1.x.y
+   ../admin/prepare-release 1.x.y
+   ```
+   This will check all of the repositories are ready to be released. This includes
+   listing any open pull-requests. The Release Manager should check nothing has been
+   missed. The script will then update the `package.json` files to reflect the
+   new version (including cross-package dependencies) and update the `CHANGELOG` files.
 
- - As much as possible of the previous steps should be converted into a CI Pipeline making use of GitHub Actions.
+   It will then raise PRs against each repository with these changes in.
 
-## Patch releases
+   **Note:** Some PRs will fail their tests at this stage because they want to
+   install versions of packages that have not been published yet in the process.
+   This is normal and will be dealt with in the next step.
+4. Run the next script in the process:
+   ```bash
+   ../admin/create-release 1.x.y
+   ```
+   For each repository in the release process it:
+    - Checks all npm dependencies have been published. If any are missing it blocks
+      until the user asks it to check again.
+    - Checks if there are any open PRs in the repository. This will include the
+      Release PR created in the previous step.
+      The Release Manager should get the Release PR (and any others that have been
+      missed at this late state) reviewed and merged before allowing the script to continue.
+      This includes ensuring the tests are passing cleanly.
+      **Note:** This will require a second person to do the review as the RM cannot
+      approve their own PRs.
+    - The script will then create a *draft* GitHub release and provide its URL.
+      The Release Manager should review the draft - check the changelog looks
+      appropriate and take note of any first-time contributors to provide to the
+      release blog post author.
+    - The script will then publish the release and move onto the next repository.
+
+### Phase Three
+
+Once everything has been published, the Release Manager should:
+
+1. Run [staging CI pipeline publish](https://github.com/flowforge/CloudProject/actions/workflows/build-kube.yml), to ensure staging is running the latest release.
```

Comment:
```suggestion
1. Run the staging CI pipeline [Build for Kube](https://github.com/flowforge/CloudProject/actions/workflows/build-kube.yml) against the main branch, to ensure staging is running the latest release.
```

Clarify the name for anyone suprised to see "Build for Kube" instead of "staging CI pipeline" when opening the link.

Reply:
_ignore duplicate suggestion (GH was acting up while doing review)_